src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -137,13 +137,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $validAssetDirs = [];
         /** @var BundleInterface $bundle */
         foreach ($kernel->getBundles() as $bundle) {
-            if (!method_exists($bundle, 'getPublicDir')) {
-                @trigger_error(sprintf('Not defining "getPublicDir()" method in the "%s" class is deprecated since Symfony 4.4 and will not be supported in 5.0.', \get_class($bundle)), E_USER_DEPRECATED);
-                $publicDir = 'Resources/public';
+            if (method_exists($bundle, 'getBundleDir') && is_dir($bundle->getBundleDir().'/public')) {
```

Comment:
you are calling `is_dir` twice on the new dir. once here and again in line 145

Reply:
I know, but I don't think it's a big deal, is it? Also, this code is meant to be removed at some point, so it would look like this at end:
```diff
-if (method_exists($bundle, 'getBundleDir') && is_dir($bundle->getBundleDir().'/public')) {
-    $originDir = $bundle->getBundleDir().'/public';
-} else {
-    $originDir = $bundle->getPath().'/Resources/public';
-}
+if (!is_dir($originDir = $bundle->getBundleDir().'/public')) {
    continue;
}
```

Reply:
However, I'd happy to change it with a better alternative that remain it clear enough for the future refactor, do you have one? Thanks!

Reply:
It will change after https://github.com/symfony/symfony/pull/32845#discussion_r311346736 ;)